src/lightning_app/storage/__init__.py

```diff
@@ -1,4 +1,36 @@
-from lightning_app.storage.drive import Drive  # noqa: F401
-from lightning_app.storage.mount import Mount  # noqa: F401
-from lightning_app.storage.path import Path  # noqa: F401
-from lightning_app.storage.payload import Payload  # noqa: F401
+from lightning_app.storage.copier import Copier, copy_files
+from lightning_app.storage.drive import Drive
+from lightning_app.storage.mount import Mount
+from lightning_app.storage.orchestrator import StorageOrchestrator
+from lightning_app.storage.path import (
+    artifacts_path,
+    filesystem,
+    is_lit_path,
+    Path,
+    path_to_work_artifact,
+    shared_storage_path,
+    storage_root_dir,
+)
+from lightning_app.storage.payload import BasePayload, Payload
+from lightning_app.storage.requests import ExistsRequest, ExistsResponse, GetRequest, GetResponse
+
+__all__ = [
```

Comment:
I think we can move most to private. Public should be `Payload, Drive, Mount, Path`

Reply:
Done :smiley: